example/RunnableDemo/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ag-demo",
+  "version": "1.0.0",
+  "description": "Demo project for the Artifact Generator",
+  "main": "index.js",
+  "author": "Andy Davidson",
```

Comment:
I guess this is a copy/paste typo, and it should be formatted according to https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors